### PR TITLE
Add cloud cost analysis takeover and engage page

### LINKED ISSUE #6872: [Webinar] [Cloud Cost Analysis] - Build takeover and engage page
## Requirements

Please build a takeover and an engage page based on this [brief](https://docs.google.com/spreadsheets/d/1Yg_sHVjaGh4snje5TltN6SFH0PDyhTEWllnaaEkKmnw/edit#gid=0) and [design](https://github.com/canonical-web-and-design/web-squad/issues/2405):
 
- [x] build takeover
- [x] build engage page
- [x] add takeover to `/takeovers/index.html`
- [x] add takeover to `/index.html`
- [x] add engage page to `/engage/index.html`
- [x] test the engage page on [https://socialdebug.com/](https://socialdebug.com/) or [http://debug.iframely.com/](http://debug.iframely.com/) and [https://cards-dev.twitter.com/validator](https://cards-dev.twitter.com/validator)

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Added private cloud cost analysis takeover and engage page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Refresh until you see the "Hosted private cloud infrastructure: a cost analysis" takeover
- Check that it matches [the brief](https://docs.google.com/spreadsheets/d/1Yg_sHVjaGh4snje5TltN6SFH0PDyhTEWllnaaEkKmnw/edit#gid=877682240) and [the design](https://github.com/canonical-web-and-design/web-squad/issues/2405)
- Click through to the engage page
- Check that it matches [the copy doc](https://docs.google.com/document/d/1Ju0tF-bvFns_91a37MP6LQ6-pTDuHqbeYI_QxXftPK0/edit), [the brief](https://docs.google.com/spreadsheets/d/1Yg_sHVjaGh4snje5TltN6SFH0PDyhTEWllnaaEkKmnw/edit#gid=877682240) and [the design](https://github.com/canonical-web-and-design/web-squad/issues/2405)
- Go to /takeovers and check that the new takeover is at the bottom of the page
- Go to /engage and check that the new engage page is at the bottom of the page
- Test the engage page on [https://socialdebug.com/](https://socialdebug.com/) or [http://debug.iframely.com/](http://debug.iframely.com/) and [https://cards-dev.twitter.com/validator](https://cards-dev.twitter.com/validator)

## Issue / Card

Fixes #6872

### CODE REVIEW

File: templates/takeovers/index.html [hunk 1]
```diff
@@ -110,4 +110,6 @@
 {% include "takeovers/_anbox-cloud-webinar.html" %}
 <p>26 Feb 2020</p>
 {% include "takeovers/_smart-start-webinar.html" %}
+<p>4th Mar 2020</p>
```
Comment (sowasred2012) [inline templates/takeovers/index.html]: It's the 2nd, or are we holding off publishing this till Weds?

Comment (steverydz) [reply templates/takeovers/index.html]: I guess so - unless we hear otherwise

**Comment by steverydz** [comment]
@anasereijo demo is up now

**Comment by anasereijo** [comment]
@steverydz for the takeover would be nice if we can make the text 7 col instead of 6. Giving a +1 anyway as I'm not sure if it's possible (or easy) to change the template.

**Comment by steverydz** [comment]
@SirSamTumless this is ready to go but the brief says the live date is 4th March so do we wait til then or can we publish now?

**Comment by anasereijo** [comment]
Alex said the following: 

>  If we push it live as a 4th takeover rather than replacing one then i am happy!

**Comment by steverydz** [comment]
OK, I'll make it live

**Pull request merged**